### PR TITLE
Add JSON Handler.  Leave pre-existing web.config.

### LINKED ISSUE #113: Default web.config file missing JSON handler.
The default `web.config` file that is now automatically created does not have a JSON handler.  This means the `content.json` file is unable to be loaded, which drives all Stache searching capabilities.

### PR DESCRIPTION
**Pull request by Blackbaud-BobbyEarl**
Fixes #113 #114

### CODE REVIEW

File: tasks/stache.js [hunk 1]
```diff
@@ -661,22 +661,56 @@ module.exports = function (grunt) {
...
+
+                    // If a file is found, stop.
+                    break;
+                }
+            }
 
             if (config.bbauth.isEnabled === true) {
-                console.log("COPYING DLL FILES...");
+                if (!template) {
+                    slog.warning("BBAUTH is enabled, but a custom web.config was found. Please convert your custom web.config to a Handlebars template (`/static/web.config.hbs`) so that the appropriate application settings can be implemented.");
```
Comment (blackbaud-brandonhare) [inline tasks/stache.js:712]: Now THAT is an error message. 👍

File: src/vendor/bbauth/web.config.hbs [hunk 2]
```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0"?>
 <!-- Web.Config Configuration File -->
 <configuration>
-  {{#if isEnabled }}
   <appSettings>
-    <add key="AuthAddress" value="{{ authAddress }}" />
-    <add key="RootUrl" value="{{ rootUrl }}" />
-    <add key="ClientId" value="{{ clientId }}" />
+    {{#if bbauth.isEnabled }}
```
Comment (blackbaud-brandonhare) [inline src/vendor/bbauth/web.config.hbs:5]: This is the check we had in here before right? Does this get generated from bbauth, or is this a stache.yml config? Just curious.

**Review by blackbaud-brandonhare — APPROVED** [review]
This looks good to me Steve.

File: src/vendor/bbauth/web.config.hbs (hunk 2 above)
Comment (Blackbaud-SteveBrush) [reply src/vendor/bbauth/web.config.hbs:5]: Stache YAML variable!

**Pull request merged**